src/components/app-configure/AppAuth.vue

```diff
@@ -28,7 +28,21 @@
         />
       </div>
       <div class="flex column">
-        <div class="flex input-group">
+        <div class="flex" style="gap: 0.625rem">
+          <span class="body-1" style="color: var(--text-grey); font-family: var(--font-title);">
+            Aggregate Login
+          </span>
+          <v-switch :value="hasAggregateLogin" @checked="updateAggregateLogin" :disabled="isConfigured" style="margin-top: 1px"></v-switch>
+          <v-tooltip
+            title="Aggregate Login (Needs content here)"
```

Comment:
Who needs to provide this content?

Reply:
Anirudh, there is a ticket, I have added the blocking dependency